docs/content/getting-started/CLI.md

```diff
@@ -0,0 +1,33 @@
+---
+category: getting-started
+categoryindex: 1
+index: 3
+---
+
+# Command Line Arguments
+
+## Example Command
+
+When running the CLI tool from the command line, the bare minimum you need to provide is the path to the project file(s) you want to analyze.
+
+```shell
+dotnet fsharp-analyzers --project ./YourProject.fsproj
+```
+
+An optional argument you may need to provide is `--analyzers-path`. This is the path to the directory containing the analyzer DLLs. 
+
+```shell
+dotnet fsharp-analyzers --project ./YourProject.fsproj --analyzers-path ./path/to/analyzers/directory
+```
+
+⚠️ If you don't provide this argument, it will default to `packages/analyzers`.
```

Comment:
This is really a Paket thing and should be mentioned as such. Folks using NuGet might not get how you can ever download the analyzers in that folder.